src/main/java/ch/naviqore/service/ArrivalTime.java

```diff
@@ -0,0 +1,9 @@
+package ch.naviqore.service;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.time.LocalDate;
+
+public interface ArrivalTime {
```

Comment:
Why do we need this? LocalDateTime and the variable name arrival or departure time should be sufficient in my opinion.